scripts/build_spack_package.sh

```diff
@@ -46,12 +46,20 @@ case $SPACK_VERSION in
 		echo "SPACK_VERSION" must be \"latest_release\" or \"develop\"
 		exit 1
 esac
+
+mkdir ~/.spack
```

Comment:
Should this be the path `~/.spack-ci` referenced in the other files?

Reply:
No, `~/.spack` is the transient configuration. `~/.spack-ci` is restored from the cache. We should probably rename it to `~/.spack-cache`.

Reply:
Thanks. `spack-cache` is a better name.